### DISCUSSION TITLE
Can't decrypt passwords

### DISCUSSION THREAD

**Discussion by HotelLima521** [zeapo/Android-Password-Store#1288] [Q&A] [answered]
Downloaded and installed OpenKeychain on Android, first generated a new keypair. Recieved the message:

Error Opening from OpenKeyChain : No encrypted data with known secret key found in stream!

I figured I should maybe just import the key pair I'm already using. I went ahead and did that, yet I'm still recieving the same problem. Perhaps Password Store doesn't know how to communicate to OpenKeyChain wrong? Or what am I doing wrong?

**Comment by msfjarvis** [comment, marked as answer]
You need to import the secret key of the keypair that you used to encrypt your password store with, into OpenKeychain. Generating a new one will not work, only importing the public part will not work.

**Reply by HotelLima521** [reply]
Right, I imported both the pubkey and private key I generated, and that I'm using from the original keychain. I'll try to delete and reimport though

**Reply by HotelLima521** [reply]
It looks like it's maybe an issue with OpenKeychain, I definitely have the right key pair on there. This is the error message I get when I click the refresh icon under the key's menu. I received this issue both on wifi and on my cell's data, so ruled out firewall and IPS from my network.

![Screenshot_20210128-021734.jpg](https://user-images.githubusercontent.com/25777722/106116634-5eee2080-610f-11eb-900e-ec534d36218f.jpg)

**Reply by msfjarvis** [reply]
Go to OpenKeychain settings  > Key Search > Manage OpenPGP keyservers and drag the sks-keyservers.net server to the top, that will hopefully fix it.

**Reply by HotelLima521** [reply]
Awesome that seemed to fix it! I also resynced my repository just in case, the two together probably did the thing. I appreciate you bud!
